### PR TITLE
changed the image to the high res version

### LINKED ISSUE #9270: Blurry Firefox Nightly logo on Nightly first run page
## Description

[1640082](https://bugzilla.mozilla.org/show_bug.cgi?id=1640082) |  Blurry Firefox Nightly logo on Nightly first run page

Details are in the bug . . .

#### Success Criteria
- [ ] Image is shown w. high-res image to all screens - high-res it's 256x256

### PR DESCRIPTION
**Pull request by SamarthAroraa**
## Issue / Bugzilla link
Closes #9270

### CODE REVIEW

File: media/css/firefox/nightly-firstrun.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ main {
     padding-bottom: 0;
 
     .mzp-c-hero-title {
-        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg.png', 124px, 124px);
+        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png', 124px, 124px);
```
Comment (craigcook) [inline media/css/firefox/nightly-firstrun.scss:23]: This mixin automatically generates CSS with the given file as a default used for all screens and a high-res file for higher resolution screens as a secondary rule in a media query. The mixin will append the "-high-res" suffix. So what you've done here is instruct it to use a file named "logo-lg-high-res-high-res.png" for high resolution displays, and that file doesn't exist so this won't work.

To fix this bug we should instead NOT use the at2x mixin and explicitly use the high-res image for all screens.

```suggestion
        background-image: url('/logos/firefox/browser/nightly/logo-lg-high-res.png');
        background-size: 124px 124px;
```

**Comment by SamarthAroraa** [comment]
Hi @craigcook ! Thanks for the review! 

- Actually what I found in the previous version of the code, with the at2x mixin used as:-

`@include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg.png', 124px, 124px);`
The image being renders was a 128 x 128px one.

Screenshot 2020-08-08 at 11 40 23 AM

- However, when I replaced it with 

`@include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png', 124px, 124px);`
The (previously 128 x 128px) image changed to a sharper 256 x 256px image and the blurriness seemed to go away. 

Screenshot 2020-08-08 at 11 40 57 AM

- I also tried to get rid of the at2x mixin altogether and replace it with  
        `background-image: url('/logos/firefox/browser/nightly/logo-lg-high-res.png');`
          `background-size: 124px 124px;`
as you suggested, but then the image didn't render altogether
Screenshot 2020-08-08 at 11 41 30 AM

Maybe, I have made a mistake. If so, I apologize in advance. I'm new to the bedrock code structure. I would really appreciate your insights on this issue!

**Comment by craigcook** [comment]
> `@include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png', 124px, 124px);`
> The (previously 128 x 128px) image changed to a sharper 256 x 256px image and the blurriness seemed to go away.

That works at lower resolutions, but at higher resolutions the CSS will reference a different image that doesn't exist. The easiest way to see this without an actual high-res screen is to switch to "responsive mode" and set the DPR to 2 or 3 to simulate one. It shows a blank space and a 404 error for the file `logo-lg-high-res-high-res.png`

image

>     * I also tried to get rid of the at2x mixin altogether and replace it with
>       `background-image: url('/logos/firefox/browser/nightly/logo-lg-high-res.png');`
>       `background-size: 124px 124px;`

Oops! My mistake, I left out the `#{$image-path}` in my suggestion. It should be `background-image: url('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png');`

**Comment by SamarthAroraa** [comment]
I made the change and it seems to work now on the simulated high-resolution screen.

Screenshot 2020-08-10 at 9 57 23 AM

File: media/css/firefox/nightly-firstrun.scss [hunk 2]
```diff
@@ -20,7 +20,8 @@ main {
     padding-bottom: 0;
 
     .mzp-c-hero-title {
-        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg.png', 124px, 124px);
+        background-image: url('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png');
+        background-size: 124px 124px;
```
Comment (craigcook) [inline media/css/firefox/nightly-firstrun.scss:24]: I'm so sorry, I have once again led you astray... it looks like the background size should actually be 80px by 80px, which is inherited from the design system so we can just delete this line as well as the padding below.

```suggestion
```

It's not your fault, the 124px size that was there originally was already wrong and I hadn't tested on small screens. This bug is currently live but only appears in small windows, though this page gets very little mobile traffic. But we can fix it now by removing the size and padding and letting the standard styling take effect.

image

File: media/css/firefox/nightly-firstrun.scss [hunk 3]
```diff
@@ -20,7 +20,8 @@ main {
...
         padding-top: 120px;
```
Comment (craigcook) [inline media/css/firefox/nightly-firstrun.scss:25]: We can delete this line.
```suggestion
```

**Comment by SamarthAroraa** [comment]
No issues @craigcook . I've committed the suggested changes.

**Pull request merged**